metrics-facade-base/src/main/java/com/ringcentral/platform/metrics/dimensions/MetricDimensionUtils.java

```diff
@@ -11,6 +11,10 @@
 @SuppressWarnings("ForLoopReplaceableByForEach")
 public class MetricDimensionUtils {
 
+    public static final MetricDimension NAME_DIMENSION = new MetricDimension("name");
```

Comment:
This class is supposed to consist of only general-purpose utility methods for working with MetricDimensions. I believe we should place these specific MetricDimensions in the corresponding specific classes in which they are used.

Reply:
ok, fixed, thank you.